### ISSUE TITLE
Add references to DITA OT Day 2022 videos

### ISSUE THREAD

**Issue by raducoravu** [dita-ot/docs#446]
Here:
https://www.dita-ot.org/dev/topics/dita-ot-day-videos.html

add new references to the DITA OT Day 2022 videos:
https://www.oxygenxml.com/events/2022/dita-ot_day.html

**Comment by lief-erickson** [comment]
A few years ago I wrote a Python script to collect the links, descriptions, and such. I'll dig up that script and update.

**Comment by infotexture** [comment]
🙇 Thanks @lief-erickson ! 👏 

I found the link to the [script Gist](https://gist.github.com/lief-erickson/4019b6ddf09c154164d4830b0fd6e7e8) in #447 and wondered if we should add that to the docs repo and/or describe its usage in the docs wiki.

On [Line 23](https://gist.github.com/lief-erickson/4019b6ddf09c154164d4830b0fd6e7e8#file-dita-ot-day-ditamap-keys-py-L23), there's a bit that refers to the years that _are/have been/should be_ scraped. 

Should we update that to include 2022, or implement the suggestion from [Line 20](https://gist.github.com/lief-erickson/4019b6ddf09c154164d4830b0fd6e7e8#file-dita-ot-day-ditamap-keys-py-L20)? 🤔

**Comment by lief-erickson** [comment]
@infotexture, I did need to add 2022, which I had done locally and have also done so in the public Gist.

This question about adding the script to the docs repo came up a few years ago. The decision we reached was to use the public Gist. The logic, as I recall, essentially was that it wasn't "core" functionality. After a couple years away from the script and having to reacquaint myself with it, I agree much more today than back then. Yes, it worked (thank you reminder comments!), but it's pretty amateur. For an every-now-and-again-and-who-knows-when-again-if-ever task, I think the public Gist is the right place. Of course we can discuss and/or revisit at any time.
